### DISCUSSION TITLE
Frequency by channel on Crazyradio

### DISCUSSION THREAD

**Discussion by WoodyloveBo** [bitcraze/discussions#1137] [Q&A]
hello.
  I'm working on a project that uses multiple Crazyflies. I understand that communication between Crazyradio and Crazyflie operates at 2.4gHz and provides channels from 1 to 100, but I would like to know how to check the specific frequency for each channel. 

Thank you.

**Comment by ataffanel** [comment]
Channels are spaced by 1MHz . So channel 0 is centered around 2400MHz and channel 100 around 2500MHz.

One thing worthy of note is that you should space communication by at least 2MHz (so 2 channels) when communicating at 2MBit/s. It is also good to have different address for all crazyflies even not on the same channel since we have seen that sometime communication leaks through channels.

**Reply by WoodyloveBo** [reply]
Thank you for answer. It was very helpful.
